test/schema.sql

```diff
@@ -179,3 +179,14 @@ CREATE TABLE messages (
   INDEX type (type),
   INDEX receipt (receipt)
 );
+
+CREATE TABLE user_space_activities (
```

Comment:
Maybe `user_activity`

Reply:
Or user_stats ?

Reply:
or user_data or user_counts 😋  `user_activity` sounds good because we can add more activity here

Reply:
PRIMARY KEY is a combination of `user` and `space`, so this table can only track space related user's activities, and only counters.

Reply:
humm,, go for anything simple or just `activity` Last option is `user_space_activity`